lib/isMobilePhone.js

```diff
@@ -67,6 +67,7 @@ var phones = {
   'es-MX': /^(\+?52)?(1|01)?\d{10,11}$/,
   'es-PA': /^(\+?507)\d{7,8}$/,
   'es-PY': /^(\+?595|0)9[9876]\d{7}$/,
+  'es-AR': /^\+?549(11|[2368]\d)\d{8}$/,
```

Comment:
Tests are failing coz you made your changes in a wrong file, you should make this in `src/lib/isMobilePhone.js`; and then this will be auto-generated when you run `npm test`. Please do that, run `npm-test` then commit and push.